src/libraries/System.Diagnostics.DiagnosticSource/tests/MetricEventSourceTests.cs

```diff
@@ -16,13 +16,16 @@ namespace System.Diagnostics.Metrics.Tests
     public class MetricEventSourceTests
     {
         ITestOutputHelper _output;
+        static readonly double s_intervalSecs = 1;
+        static readonly TimeSpan s_waitForEventTimeout = TimeSpan.FromSeconds(60);
```

Comment:
Nits:
- These should be explicitly `private`
- s_intervalSecs should be a const rather than static readonly, and should be named IntervalSecs

Reply:
fixed